FormSubmissionAndValidation2/src/components/TextAreaBox.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
```

Comment:
For shared components like this, we usually have a directory called shared or common where the reuseable, non-flow specific components are kept. 

For your reference, Specific to SM, we have a ui package where we have designed and stored ui elements to be used across every vertical.

Reply:
We also prefer to use hyphenated file/folder names instead of snake or camel case. text-area-box instead of TextAreaBox

Reply:
sure will make this change